src/elm/App/Update.elm

```diff
@@ -21,6 +21,10 @@ type Action
   -- NoOp actions
   | NoOp
 
+dragActions : Signal Action
+dragActions =
+    Signal.map ChildTimelineAction Timeline.Update.startTimeActions
```

Comment:
If you've got multiple components that each get their own reference (i.e. they are individually named in the code, not in a List), then I'd probably just add an `Action` tag for each of them, and then the `dragActions` would end up being a `mergeMany` of mapping the actions to the tags.

So, suppose you had a second timeline (but only two ... not an arbitrary number) ... then in `update` you'd have a `ChildTimelineAction2` case, and in `Action` you'd have a `ChildTimelineAction2` tag, and `dragActions` would look something like:

```elm
dragActions : Signal Action
dragActions =
    Signal.mergeMany
        [ Signal.map ChildTimelineAction Timeline.Update.startTimeActions
        , Signal.map ChildTimelineAction2 Timeline.Update.startTimeActions2
        ]
```

Or something like that -- I'm just thinking out loud, so there may be a flaw there.

If you need to keep an arbitrary number of draggable things in a `List`, then one would have to switch to the `trackMany` approach and then supply an index that gets threaded to the right place -- the threading would be similar, but the action tags would have some kind of index associated with them, so you can pick the item out of the list. (Sort of like how in the Elm Architecture Tutorial when they switch from "two counters" to "a list of counters").